metrics/cli/lib/sentry/cli/sentry_cli.dart

```diff
@@ -14,38 +15,38 @@ class SentryCli extends Cli {
     return run(['login']);
   }
 
-  /// Creates a Sentry release with the given [releaseName]
-  /// within the given [project].
-  Future<void> createRelease(String releaseName, SentryProject project) {
+  /// Creates a Sentry release with the given [release].
+  Future<void> createRelease(SentryRelease release) {
+    final project = release?.project;
```

Comment:
Maybe it would be better to throw an ArgumentError if the given `release` is null?